akka-stream/src/main/scala/akka/stream/impl/Stages.scala

```diff
@@ -30,6 +30,7 @@ import akka.stream._
     val mapAsyncUnordered = name("mapAsyncUnordered")
     val grouped = name("grouped")
     val groupedWithin = name("groupedWithin")
```

Comment:
actually, it is:
https://github.com/akka/akka/pull/22712/files#diff-d25227230bef7fd44d678521a8f07f1cR993